charts/seed-controlplane/charts/kube-apiserver-hpa/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Why do we need a separate chart for this? Let's please put the `kube-apiserver-hpa.yaml` into the `kube-apiserver` chart and control whether to enable/disable it via the `values.yaml` there.